### DISCUSSION TITLE
wire:navigate.hover cache should be temporary

### DISCUSSION THREAD

**Discussion by stancl** [livewire/livewire#7943] [3. Feature ideas]
The main use case for using `wire:navigate.hover` instead of just `wire:navigate` is when you want to provide the response to the user *even* faster, and the marginal increase in server load isn't a concern. I think this description fits a lot of applications.

Many of those applications, however, may have quickly changing state on certain pages, and it seems that once Livewire populates the prefetch cache with a given page, the cache isn't ever pruned, so a new hover never triggers another request unless the page is refreshed.

I think it'd make sense to provide some configurable TTL for this cache. Maybe some global value, configurable either in PHP or JS, or a component-specific value, perhaps configured using an attribute like `#[PrefetchTTL(60)]`, that could add this information to the response in some way (since it's similar to cache headers, a header might work well here).

**Comment by kohle** [comment]
Hi, does anyone know if there is a workaround or alternative for this?

**Comment by yamakrane** [comment]
Is there a way to disable cache for wire:navigate.hover?

**Comment by A10-tomi** [comment]
I'm not sure this even needs to be configurable. I don't see any point in keeping data in cache after the first load following the prefetch.

This creates a significant issue for apps with frequently updating content, as users end up with stale data on subsequent page loads, leading to a poor user experience.
